b9lab-content/4-running-a-chain/04-node-api-and-cli.md

```diff
@@ -1 +1,292 @@
+```
+
+In the output you can see that your chain_id is test-chain-rT4wZY. We could determine the chainid by passing the flag --chain-id.
+
+Inspect the initial configuration:
+
+```bash
+$ cat ~/.simapp/config/genesis.json 
+```
+
+Inspect your keys:
+
+```bash
+$ ./simd keys list
+[]
+
+```
+
+As you might have expected, you do not have any keys yet. Add a new key:
+
+```bash
+$ ./simd keys add b9lab
+
+- name: b9lab
+  type: local
+  address: cosmos1nw793j9xvdzl2uc9ly8fas5tcfwfetercpdfqq
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A6TrsRO/OH91fAEFLohw7RwFB832NRsRWhQvE2t8cfLK"}'
+  mnemonic: ""
+
+
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+ivory uniform actual spot floor vessel monster rose yellow noise smile odor veteran human reason miss stadium phrase assault puzzle sentence approve coral apology
+```
+
+You can see the mnemonic at the end of the output:
+```bash
+ivory uniform actual spot floor vessel monster rose yellow noise smile odor veteran human reason miss stadium phrase assault puzzle sentence approve coral apology
+```
+
+Tis sequence of words is a mnemonic you can use to recover your public and private keys. In a production setting, the mnenonic should be stored in a reliable and confidential fashion such as a key-management infrastructure. 
+
+Confirm that the key is has been added:
+
+```bash
+$ ./simd keys list
+```
+
+or with:
+
+```bash
+$ ./simd keys show b9lab
+```
+
+A cosmos-sdk blockchain relies on validators to produce blocks. Validators are normally added and removed through an on-chain governance process, but initially there is no validator to generate the block in which such a transaction might exist. Your chain needs an account for bootstrapping purposes. 
+
+Add your key (also know as an account) to the genesis file:
+
+```bash
+$ ./simd add-genesis-account b9lab 100000000stake
+```
+
+The suffix, stake appended to the amount and configured in the genesis file is the unit for the tokens in this chain. This command `100000000` to your account. Check it with:
+
+```bash
+$ cat ~/.simapp/config/genesis.json | grep -A 2 -B 2 stake
+```
+
+Because your blockchain is not actually running yet, you need to include bootstrap transactions in the genesis file.
+
+
+Note: In this scenario, you will need the 2/3 threshold for validation, so you will stake 70000000stake of our 100000000stake in the b9lab account we just created. 
```

Comment:
```suggestion
Note: In this scenario, you must meet the 2/3 threshold for validation, so you must stake 70000000stake of your 100000000stake in the b9lab account you just created. 
```

Reply:
see how tricky it gets with different audiences (we or you)
be consistent and always use "you" as the second person audience